Source/Launcher/Program.cs

```diff
@@ -37,59 +36,116 @@ namespace ORTS
 {
 	static class Program
 	{
-		/// <summary>
-		/// The main entry point for the application.
-		/// </summary>
-		[STAThread]
-		static void Main()
+        static string downloadUrl;
+        static string downloadText;
+        static List<string>missing = new List<string>();
```

Comment:
This seems like an odd way to approach this. It would be clearer to have a struct with three fields (name, downloadText, downloadUrl) and make the list contain those.